hueniversitypy/theme_ubc.py

```diff
@@ -1,21 +1,58 @@
-def theme_ubc(palette='blue'):
+def theme_ubc():
     """ 
     Applies a University of British Columbia theme to all subsequential altair plot objects so they are displayed with the UBC visual identity. 
-    Two palettes based on the UBC visual identity guidelines can be selected: 'blue' and 'white'. 
-	
-    Parameters
-    ----------
-    palette : 'blue' or 'white'
-    	specifies which colour palette defined in the visual identity documentation is used. 
-	
+    See the visual identity at https://brand.ubc.ca/guidelines/downloads/ubc-colour-and-fonts/ 
+		
     Returns
     -------
     altair plot : altair.vegalite.v4.api.Chart
-    	an altair plot with the UBC visual identity colour theme applied. 
+    an altair plot with the UBC visual identity colour theme applied. 
 
     Example
     ----------
     >>> alt.themes.register('theme_ubc', theme_ubc)
     >>> alt.themes.enable('theme_ubc')
-
     """
+    #attribution to Sergio Sanchez 
+    #https://towardsdatascience.com/consistently-beautiful-visualizations-with-altair-themes-c7f9f889602
+
+    font = "Arial"
+    labelFont = "Arial" 
+    sourceFont = "Arial"
+    # Axes
+    axisColor = "#000000"
+    gridColor = "#DEDDDD"
+    # Colors
+    main_palette = ['#002145', '#0055B7', '#00A7E1', '#40B4E5', '#6EC4E8', '#97D4E9']
```

Comment:
```suggestion
    ubc_palette = ['#002145', '#0055B7', '#00A7E1', '#40B4E5', '#6EC4E8', '#97D4E9']
```
Just changed palette name so it's clear which one we're using.

Reply:
If I change the name here I will have to change below as well. Please accept this PR I will modify it in the next edit along with renaming `test_plot` to `ubc_plot`

Reply:
Ah good point.